VideoPost/VideoPost/CameraController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CameraController.swift
+//  VideoPost
+//
+//  Created by Hunter Oppel on 6/4/20.
+//  Copyright © 2020 LambdaSchool. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+class CameraController {
+    
+    lazy var captureSession = AVCaptureSession()
+    lazy var fileOutput = AVCaptureMovieFileOutput()
+    
+    func setUpCaptureSession() {
+        captureSession.beginConfiguration()
+        
+        let camera = bestCamera()
+        guard let cameraInput = try? AVCaptureDeviceInput(device: camera),
+            captureSession.canAddInput(cameraInput) else {
+                fatalError("")
+        }
```

Comment:
1. You checked about adding the camera, but never added it as input.

2. If you look in your recordings folder you'll have some audio recordings without video. Don't forget to check on the device using Devices & Simulators.

Add

```swift
captureSession.addInput(cameraInput)
```